lib/generators/decidim/app_generator.rb

```diff
@@ -73,6 +77,8 @@ def readme
       end
 
       def gemfile
+        return unless options[:skip_gemfile]
```

Comment:
It's `if options[:skip_gemfile]` :smiley:

Reply:
Seen! Damn, what a day